platforms/documentation/docs/src/docs/userguide/dep-man/understanding-dependency-resolution/variant_attributes.adoc

```diff
@@ -205,32 +301,43 @@ Registering an attribute with the schema is required in order to use Compatibili
 
 Each configuration has a container of attributes.  Attributes can be configured to set values:
 
-.Setting attributes on configurations
```

Comment:
There is no anchors for that injected, right?

Reply:
No anchors.